google-datacatalog-rdbms-connector/src/google/datacatalog_connectors/rdbms/prepare/datacatalog_tag_factory.py

```diff
@@ -104,6 +104,43 @@ def make_tag_for_table_metadata(self, tag_template, table,
 
         return tag
 
+    def make_tags_for_columns_metadata(self, tag_template, table):
+        """
+         Create Tags for the Table Columns technical fields that
+          aren't support yet by Datacatalog api.
+
+         :param tag_template: A datacatalog.TagTemplate()
+         :param table:
+         :return: list[tag]
+        """
+
+        tags = []
+
+        columns = table.get('columns')
+
+        if columns:
```

Comment:
Nesting the `for` loop to the `if` statement can be avoided to improve readability. WDYT about the below code?
```
columns = table.get('columns')

if not columns:
    return []

tags = []
for column in columns:
   ....
```